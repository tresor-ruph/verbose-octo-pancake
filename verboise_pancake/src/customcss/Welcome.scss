.main-welcome {
    width: 100%;
    height: 100vh;
    background-color: white;   
}

.content {
    background-color: white;
    margin-top: 25vh;
    width: 50vw;
    height: 40vh;
    text-align: center;
}
.welcome-title {
    font-size: 2vw;
    color: #888;
    font-weight: 500;

}
.welcome-sub-content {
    margin-top: 8vh;
}
.welcome-content {
    margin-top: 2vh;
    font-size: 1.2vw;
}
.start-btn {
    margin-top: 2vh;
}
.create-Event-btn {
    background-color: #00C0F8;
    color: white;
}
