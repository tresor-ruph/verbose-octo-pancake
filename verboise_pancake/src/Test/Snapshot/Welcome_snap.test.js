import React from 'react'
import renderer from 'react-test-renderer'
import Welcome  from './../../components/Home/WelcomeMessage'

it('renders correctly', () => {
    const WelcomeSanp = renderer.create(<Welcome />).toJSON();
    expect(WelcomeSanp).toMatchSnapshot();

});