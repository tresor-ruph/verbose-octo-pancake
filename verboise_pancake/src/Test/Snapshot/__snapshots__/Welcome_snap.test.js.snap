// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="home"
  >
    <div
      className="card content p-shadow-23"
    >
      <span
        className="welcome-title"
      >
        Welcome undefined  !
      </span>
      <div
        className="welcome-content"
      >
        Create your first event now and include your audience
      </div>
      <div
        className="start-btn"
      >
        <button
          className="test btn btn-primary btn-lg"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          <span
            style={
              Object {
                "fontSize": "1.1rem",
                "fontWeight": "bold",
              }
            }
          >
            Create your first Event
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
